Resources/Storage/GFAL2_StorageBase.py

```diff
@@ -710,6 +713,9 @@ def __getSingleMetadata( self, path ):
           metadataDict['Unavailable'] = 1
         else:
           metadataDict['Unavailable'] = 0
+        if attributeDict['user.status'] == 'ONLINE_AND_NEARLINE':
```

Comment:
Ah you are right, there is no online status for xroot. I will move that part to the SRM module and extend the XROOT module with the hardcoded version?